invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/components/DeleteRecordModal/DeleteRecordModal.js

```diff
@@ -89,7 +96,8 @@ export default class DeleteRecordModal extends Component {
 
   renderAll() {
     const { data, refProps } = this.props;
```

Comment:
`data` missing from `propTypes` of this component (`isLoading` and `hasError` too)